test/integration/v2/sql.test.ts

```diff
@@ -17,7 +17,8 @@ const connectionParams = {
 jest.setTimeout(20000);
 
 describe("sql queries are supported", () => {
-  it("supports use database statement", async () => {
+  // TODO: Unskip when this is supported on staging
+  it.skip("supports use database statement", async () => {
     const new_database_name =
```

Comment:
Can we add here an expected failure? This way once it's fixed in STG we'll see tests failing and fix them.